components/script_traits/lib.rs

```diff
@@ -443,11 +443,11 @@ pub struct TouchId(pub i32);
 #[derive(Clone, Copy, Debug, Deserialize, Serialize)]
 pub enum MouseButton {
     /// The left mouse button.
-    Left,
-    /// The middle mouse button.
-    Middle,
+    Left = 1,
     /// The right mouse button.
-    Right,
+    Right = 2,
+    /// The middle mouse button.
+    Middle = 4,
```

Comment:
Shouldn't this use a new MouseButtonMask enum, rather than mangling this one?

Reply:
I could add a new enum for this purpose, but it would be exactly the same as this one since `MouseButtonEvent` (which I use for updating the `buttons` bitmask) stores the button that triggered the event as a `MouseButton`.

Reply:
Then don't alter the values of this enum, just map them ala `MouseButton::Left => 1` etc.